pl/main.lang

```diff
@@ -830,11 +830,11 @@ Przesyłane będą wyłącznie informacje związane z Mozillą.
 
 
 ;Thanks!
-Thanks!
+Dziękujemy!
 
 
 ;If you haven’t previously confirmed a subscription to a Mozilla-related newsletter you may have to do so. Please check your inbox or your spam filter for an email from us.
-If you haven’t previously confirmed a subscription to a Mozilla-related newsletter you may have to do so. Please check your inbox or your spam filter for an email from us.
+Jeśli jeszcze nigdy nie potwierdzono subskrypcji newslettera Mozilli, to musisz to zrobić teraz. Sprawdź, czy w skrzynce pocztowej lub spamie jest wiadomość od nas.
```

Comment:
Not "must to" but "may have to" and there is no time condition, so maybe something like "to możesz musieć to zrobić"